Index: src/haftalikTaskler/T03_TasKagitMakas.java
===================================================================
diff --git a/src/haftalikTaskler/T03_TasKagitMakas.java b/src/haftalikTaskler/T03_TasKagitMakas.java
deleted file mode 100644
--- a/src/haftalikTaskler/T03_TasKagitMakas.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,56 +0,0 @@
-package haftalikTaskler;
-
-import java.util.Scanner;
-
-public class T03_TasKagitMakas {//verildi
-
-    public static void main(String[] args) {
-        // tas >makas
-        // makas >kagit
-        //kagit >tas
-        int numberOfFirstWin = 0;
-        int numberOfSecondWin = 0;
-        int numberOfComWin = 0;
-        int gameOver = 3;
-
-//		List <String> elemanlar = new ArrayList<>();
-//		elemanlar.add("Tas");
-//		elemanlar.add("Makas");
-//		elemanlar.add("Kagit");
-        Scanner scan = new Scanner(System.in);
-
-        while (numberOfFirstWin < gameOver && numberOfComWin < gameOver) {
-            System.out.println("1. Oynucu Secimi: \nTas secmek icin 0 \n Makas secmek icin 1 \n Kagit secmek icin 2 \n seciniz.");
-            int myGuess = scan.nextInt();
-            System.out.println("2. Oynucu Secimi: \nTas secmek icin 0 \n Makas secmek icin 1 \n Kagit secmek icin 2 \n seciniz.");
-
-            int yourGuess = scan.nextInt();
-            ;
-            System.out.println("1. Oyuncu Tahmini:" + myGuess + "\n" + "2. Oyuncu Tahmini:" + yourGuess);
-
-            if (myGuess == yourGuess) {
-                System.out.println("Berabere");
-            } else if ((myGuess == 0 && yourGuess == 1) || (myGuess == 1 && yourGuess == 2) || (myGuess == 2 && yourGuess == 0)) {
-                System.out.println("1. oyuncu Kazandi");
-                numberOfFirstWin++;
-            } else {
-                System.out.println("2. oyuncu Kazandi");
-                numberOfComWin++;
-            }
-
-        }
-
-        if (numberOfFirstWin == 3) {
-            System.out.println("==GAME OVER==");
-            System.out.println("Sizin kazandiniz: " + numberOfFirstWin);
-            System.out.println("Kabettiniz Bilgisayarin kazanma sayisi: " + numberOfComWin);
-        } else if (numberOfComWin == 3) {
-            System.out.println("==GAME OVER==");
-            System.out.println("Biligisayar kazandi: " + numberOfComWin);
-            System.out.println("Kaybettiniz Kazandiginiz ouyn sayisi : " + numberOfFirstWin);
-        }
-
-
-    }
-
-}
Index: src/haftalikTaskler/T02_IkininKuvveti.java
===================================================================
diff --git a/src/haftalikTaskler/T02_IkininKuvveti.java b/src/haftalikTaskler/T02_IkininKuvveti.java
deleted file mode 100644
--- a/src/haftalikTaskler/T02_IkininKuvveti.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,16 +0,0 @@
-package haftalikTaskler;
-
-public class T02_IkininKuvveti {
-    public static void main(String[] args) {
-        /*
-         * Kullanicidan gelen datayi methodda parametre olarak alan
-         * ve datanın 2'nin kuvveti olup olmadığını kontrol eden PowersofTwo(num) methodu create ediniz.
-         */
-
-
-
-
-    }
-
-}
-
Index: src/haftalikTaskler/T09_ArrayList.java
===================================================================
diff --git a/src/haftalikTaskler/T09_ArrayList.java b/src/haftalikTaskler/T09_ArrayList.java
deleted file mode 100644
--- a/src/haftalikTaskler/T09_ArrayList.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,20 +0,0 @@
-package haftalikTaskler;
-
-public class T09_ArrayList {
-
-    /*
-      Task:
-
-    Create 2D ArrayList which can store String ArrayLists
-    Create 3 ArrayLists which are Employees, Employers, Companies
-    Store this 3 ArrayList in 2D ArrayList
-    String data type ArrayList'leri depolayabilen 2D ArrayList create ediniz
-    Çalışanlar, İşverenler, Şirketler olmak üzere 3 ArrayListesi oluşturun
-    Bu 3 ArrayList'i 2D ArrayList'te saklayın
-     */
-
-    public static void main(String[] args) {
-
-
-    }
-}
Index: src/haftalikTaskler/T08_ArrayList.java
===================================================================
diff --git a/src/haftalikTaskler/T08_ArrayList.java b/src/haftalikTaskler/T08_ArrayList.java
deleted file mode 100644
--- a/src/haftalikTaskler/T08_ArrayList.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,14 +0,0 @@
-package haftalikTaskler;
-
-public class T08_ArrayList {
-    public static void main(String[] args) {
-        /*
-        Bir tamsayı listesinde en yakın 2 tamsayıyı bulunuz.
-         Örnek: (12, 31, 15, 32, 54) ==> Çıktı 31 ve 32'tür
-         */
-
-
-
-            }
-
-}
Index: src/haftalikTaskler/T05_KullaniciSayiGir.java
===================================================================
diff --git a/src/haftalikTaskler/T05_KullaniciSayiGir.java b/src/haftalikTaskler/T05_KullaniciSayiGir.java
deleted file mode 100644
--- a/src/haftalikTaskler/T05_KullaniciSayiGir.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,16 +0,0 @@
-package haftalikTaskler;
-
-public class T05_KullaniciSayiGir {
-
-	public static void main(String[] args) {
-		/*
-		 * Kullanıcıya sayı girmesini söyleyin. Kullanıcı sıfır girdiğinde,
-		 * ekrana o ana kadar girmiş olduğu
-		 * tüm sayıların toplamını yazdırınız.
-		 * Ornegin; 5, 4, 7, 0 ise 5+4+7=16 oldugundan ekrana 16 yazdırın
-		 */
-
-
-
-	}
-}
Index: src/haftalikTaskler/T01_TopZıplamaMesafeSaysı.java
===================================================================
diff --git a/src/haftalikTaskler/T01_TopZıplamaMesafeSaysı.java b/src/haftalikTaskler/T01_TopZıplamaMesafeSaysı.java
deleted file mode 100644
--- a/src/haftalikTaskler/T01_TopZıplamaMesafeSaysı.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,18 +0,0 @@
-package haftalikTaskler;
-
-public class T01_TopZıplamaMesafeSaysı {
-
-   /* HAFTANIN TASK'iiii :-)
-    Bir top belirli yükseklikten atılmaktadır.
-    Atıldıktan sonra, atıldığı yüksekliğin 3/4 u kadar yerden yukarı doğru zıplamaktadır
-    Top zıplama  yüksekliği 1 metrenin altına indiğinde durmaktadır.
-    Bu ana kadar aldigi toplam yolu ve yere vurma sayısını bulan do while code blogu create ediniz.
-
-    */
-
-    public static void main(String[] args) {
-
-
-//
-    }
-}
Index: src/haftalikTaskler/T07_RomaRakamlari.java
===================================================================
diff --git a/src/haftalikTaskler/T07_RomaRakamlari.java b/src/haftalikTaskler/T07_RomaRakamlari.java
deleted file mode 100644
--- a/src/haftalikTaskler/T07_RomaRakamlari.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,53 +0,0 @@
-package haftalikTaskler;
-
-public class T07_RomaRakamlari {
-    /*  TASK :
-		 	Create a method to convert Roman numerals to numbers
-		 	Romen rakamlarını sayılara dönüştürmek için bir method create ediniz
-		 	Input:
-		 		System.out.println(romanNumbers(805));
-		   		System.out.println(fromRomans("DCCCV"));
-		   	Output: DCCCV   805
-
-
-		   		M + [C M] + [X L] + V   ---->   1945
-		   		The chars in the brackets are in ascending order.
-		   		 Parantez içindeki karakterler artan sıradadır (C-100, M-1000) --->
-		   		 [C M]=900
-
-		   		 Roma rakamları, rakamlar yan yana getirilerek ve rakam değerleri
-		   		  toplanarak yazılır. Örneğin 1 sayısı I şeklinde, 2 sayısı II
-		   		  şeklinde, 3 sayısı III şeklinde yazılır. Ancak en fazla 3
-		   		  tane aynı rakam yan yana gelebilir. Örneğin 4 sayısını IIII
-		   		   şeklinde yazamayız. 4 sayısını yazmak için bir tane 5 ve
-		   		   soluna 1 yazarız. Yani IV. Buradan da anlaşılabileceği gibi
-		   		   eğer küçük olan rakam büyük rakamın solundaysa büyük rakamdan
-		   		   çıkarılır. 6 sayısı VI, 7 sayısı VII, 8 sayısı VIII şeklinde
-		   		   yazılır. Buna göre de eğer küçük rakam büyük rakamın sağındaysa
-		   		   büyük rakamla toplanır.
-		*/
-    public static void main(String[] args) {
-
-
-
-}
-
-
-
-/*
-
-public class RomanNumeralConverter {
-    private static final HashMap<Character, Integer> romanToDecimal = new HashMap<>();
-    static {
-        romanToDecimal.put('I', 1);
-        romanToDecimal.put('V', 5);
-        romanToDecimal.put('X', 10);
-        romanToDecimal.put('L', 50);
-        romanToDecimal.put('C', 100);
-        romanToDecimal.put('D', 500);
-        romanToDecimal.put('M', 1000);
-    }
-
- */
-
-}
Index: src/haftalikTaskler/Cache.java
===================================================================
diff --git a/src/haftalikTaskler/Cache.java b/src/haftalikTaskler/Cache.java
deleted file mode 100644
--- a/src/haftalikTaskler/Cache.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,32 +0,0 @@
-package haftalikTaskler;
-
-public class Cache {
-
-    public static void main(String[] args) {//harakiri yaptiran soru
-
-
-        Integer a = 9;
-        Integer b = 9;
-
-
-        if (a == b) {
-            System.out.println("a==b");
-        } else {
-            System.out.println("a!=b");
-        }
-
-        Integer x = 396;
-        Integer y = 396;
-
-
-        if (x == y) {
-            System.out.println("x==y");
-        } else {
-            System.out.println("x!=y");
-        }
-    }
-}
-
-
-
-
Index: src/haftalikTaskler/T10_ArrayList.java
===================================================================
diff --git a/src/haftalikTaskler/T10_ArrayList.java b/src/haftalikTaskler/T10_ArrayList.java
deleted file mode 100644
--- a/src/haftalikTaskler/T10_ArrayList.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,12 +0,0 @@
-package haftalikTaskler;
-
-public class T10_ArrayList {
-    public static void main(String[] args) {
-        /* Listedeki 8 ve 12 dışındaki her öğenin değerini 2'şer artırınız.
-     Örnek: (15, 31, 8, 13, 12) ==> Çıktı (17, 33, 8, 15, 12)
-     */
-
-
-
-    }
-}
Index: src/haftalikTaskler/T11_ReversingStringCodes.java
===================================================================
diff --git a/src/haftalikTaskler/T11_ReversingStringCodes.java b/src/haftalikTaskler/T11_ReversingStringCodes.java
deleted file mode 100644
--- a/src/haftalikTaskler/T11_ReversingStringCodes.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,13 +0,0 @@
-package haftalikTaskler;
-
-
-public class T11_ReversingStringCodes {
-
-    public static void main(String[] args) {
-
-
-
-
-
-    }
-}
Index: src/haftalikTaskler/T04_AnagramKontrol.java
===================================================================
diff --git a/src/haftalikTaskler/T04_AnagramKontrol.java b/src/haftalikTaskler/T04_AnagramKontrol.java
deleted file mode 100644
--- a/src/haftalikTaskler/T04_AnagramKontrol.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,16 +0,0 @@
-package haftalikTaskler;
-
-public class T04_AnagramKontrol {
-
-    public static void main(String[] args) {
-		/*
-		 * Anagram, bir kelimenin harflerinden baska bir kelime olusturmaya denir.
-			Kullanıcıdan alınacak  iki kelimenin Anagram kontrolu yapan  Java code create ediniz
-			isAnagram("listen", "Silent") ==> true
-		 */
-
-
-    }
-
-
-}
Index: src/haftalikTaskler/T06_Arrays.java
===================================================================
diff --git a/src/haftalikTaskler/T06_Arrays.java b/src/haftalikTaskler/T06_Arrays.java
deleted file mode 100644
--- a/src/haftalikTaskler/T06_Arrays.java	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
+++ /dev/null	(revision 8211129a73d62d79d1576fafef6fbd1261dd6657)
@@ -1,18 +0,0 @@
-package haftalikTaskler;
-
-public class T06_Arrays {
-    public static void main(String[] args) {
-        /* TASK :
-         kullanıcıdan alınan bir  String'deki herbir  karakterin adedini bir array içinde yazdıran bir java programı yazın.
-         String  "Javacilar cook afilli" ise cevap şöyle olmalıdır:
-         { =2, a=3, r=1, c=1, v=1, o=2, f=1, l=2, J=1, i=2}
-         */
-
-
-
-
-    }
-
-}
-    
-
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_16\" project-jdk-name=\"openjdk-20\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision a39c024e6e3c9ff4de26175b9ff03b22705aecbe)
+++ b/.idea/misc.xml	(date 1688421766555)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_16" project-jdk-name="openjdk-20" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_20" default="true" project-jdk-name="openjdk-20" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
